tests/usdt_sample/scripts/latency.c

```diff
@@ -0,0 +1,127 @@
+#include <linux/blkdev.h>
+#include <uapi/linux/ptrace.h>
+
+// The following methods were taken from the verbose output of argdist.py.
+// Function names were adjusted to match the BPF function name expectations.
+static inline int _bpf_readarg_trace_operation_start_1(struct pt_regs* ctx, void* dest, size_t len)
```

Comment:
I think you should not define function _bpf_readarg_trace_operation_start_1 like this in your bpf program. The bcc should generate this for you. You can use debug=4 option in BPF class constructor to dump out the processed file.